pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -12,6 +12,10 @@
 , unpaper
 , liberation_ttf
 , fetchFromGitHub
+
+# Allow propagating additional dependencies through an override with
+# the packages being considered in the PYTHONPATH exposed via passthru.
+, extraPropagatedBuildInputs ? ps: []
```

Comment:
This is a mouthful, but on on point. Could be shortend to `extraPythonPackages` or even `extraPackages`.